CHANGES.rst

```diff
@@ -71,7 +72,7 @@ astropy.time
 
 astropy.units
 ^^^^^^^^^^^^^
-
+-Added function ``quantity``allclose to quantity
```

Comment:
Again, formatting is wrong (need space and empty line after, plus link to PR), and the description is too terse to be useful:
```
Added a units-aware ``allclose`` function (this was previously available in the
``tests`` module as ``quantity_allclose``) [#7252]
```